File: Python/Pow(x, n).py
def power(base, exponent):
    if exponent == 0:
        return 1.0
    elif exponent < 0:
        base = 1.0 / base
        exponent = -exponent

    result = 1.0
    for i in range(exponent):
        result *= base
    return result

base = 2.0
exponent = 3

result = power(base, exponent)

print(f"{base} raised to the power of {exponent} is {result}")

File: JS/Pow(x, n).js
function power(base, exponent) {
    if (exponent === 0) {
        return 1.0;
    } else if (exponent < 0) {
        base = 1.0 / base;
        exponent = -exponent;
    }

    let result = 1.0;
    for (let i = 0; i < exponent; i++) {
        result *= base;
    }
    return result;
}

const base = 2.0;
const exponent = 3;

const result = power(base, exponent);

console.log(`${base} raised to the power of ${exponent} is ${result}`);
